dom/nodes/insertion-effects/Node-appendChild-script-and-custom-from-fragment.tentative.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and custom element from a DocumentFragment</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+let customConstructed = false;
+let isCustomConstructed = false;
+class CustomElement extends HTMLElement {
+    constructor() {
+        super();
+        customConstructed = true;
+    }
+}
+test(() => {
+  const script = document.createElement("script");
+  script.textContent = `
+    customElements.define("custom-element", CustomElement);
+    isCustomConstructed = customConstructed;
+`;
+  const custom = document.createElement("custom-element");
+  const df = document.createDocumentFragment();
+  df.appendChild(script);
+  df.appendChild(custom);
+  assert_false(customConstructed);
+  assert_false(isCustomConstructed);
+  document.head.appendChild(df);
+  assert_true(customConstructed);
+  assert_true(isCustomConstructed);
+});
```

Comment:
I don't know _too_ much about custom elements, so pardon my ignorance. I assume this test is supposed to pass because `define()` is supposed to crawl the DOM looking for elements that have already been inserted and upgrade them to run with the given class passed into `define`? That is, I assume `custom` here is an ["upgrade candidate"](https://html.spec.whatwg.org/multipage/custom-elements.html#element-definition:~:text=Let%20upgrade%20candidates%20be%20all%20elements) at the time `define()` runs?

If that's the case, maybe just including a test description here would be useful. I.e., making this line:

```
}, "<explanation>");
```